### PR TITLE
passing round sizes to math module

### PR DESCRIPTION
**Pull request by umbernhard**
**Description**

Passes round sizes to minerva.py.

**Testing**
N/A

**Progress**

Ready.

### CODE REVIEW

File: server/api/rounds.py [hunk 1]
```diff
@@ -198,6 +198,12 @@ def cumulative_batch_results(election: Election) -> BatchTallies:
     }
 
 
+def round_sizes(election: Election) -> Dict[int, int]:
+    return {
+        rnd.round_num: len(list(rnd.sampled_ballot_draws)) for rnd in election.rounds
```
Comment (nealmcb) [inline server/api/rounds.py:203]: If the round_num values are always just counting up, would it make it easier to just return a simple List[int]? Then we don't have to argue whether the first round is 0 or 1 ;)

File: server/audit_math/ballot_polling.py [hunk 2]
```diff
@@ -19,6 +19,7 @@ def get_sample_size(
     risk_limit: int,
     contest: Contest,
     sample_results: Optional[Dict[str, Dict[str, int]]],
+    round_sizes: Dict[int, int],
```
Comment (nealmcb) [inline server/audit_math/ballot_polling.py:22]: It seems more intuitive to me to pass the round sizes before the sample results (mirroring the sequence of events in the audit procedure) but of course it matters little.

**Review by nealmcb — APPROVED** [review]
Thanks for the quick work!

File: server/api/rounds.py (hunk 1 above)
Comment (jonahkagan) [reply server/api/rounds.py:203]: I like it being explicit here, for actually a similar reason - there is no "round 0".

**Review by jonahkagan — APPROVED** [review]

File: server/api/rounds.py (hunk 1 above)
Comment (jonahkagan) [reply server/api/rounds.py:203]: Also If you wanted to do this in one database query, you could. slightly more efficient, but not a big deal either way

```py
return dict(
  Round.query.filter_by(election_id=election.id)
  .join(SampledBallotDraw).group_by(Round.id)
  .values(Round.round_num, func.count(SampledBallotDraw.ticket_number))
)
```

Comment (umbernhard) [reply server/api/rounds.py:203]: Yeah, I'll use the db query, just couldn't quite figure out how to make it work.

**Pull request merged**